tests/frontend/specs/helper.js

```diff
@@ -82,21 +92,39 @@ describe("the test helper", function(){
             // helper function seems to have cleared cookies
             // NOTE: this doesn't yet mean cookies were cleared effectively.
             // We still need to test below that we're in a new session
-            expect(window.document.cookie).to.not.be(testCookie)
-            expect(window.document.cookie).to.not.be(firstCookie)
+            expect(window.document.cookie).to.not.contain('token=foo');
+            expect(window.document.cookie).to.not.contain('language=bar');
+            expect(chrome$.document.cookie).to.contain('prefsHtml=baz');
+            expect(window.document.cookie).to.not.contain('prefsHtml=baz');
+
+            expect(window.document.cookie).to.not.be(firstCookie);
 
             // click on the settings button to make settings visible
             var $userButton = chrome$(".buttonicon-showusers");
             $userButton.click();
 
             // confirm that the session was actually cleared
             var $usernameInput = chrome$("#myusernameedit");
-            expect($usernameInput.val()).to.be('')
+            expect($usernameInput.val()).to.be('Enter your name');
```

Comment:
Not sure how this was passing before. This text looks like it was introduced long ago. Maybe we recently fixed a bug that was keeping it from appearing?

Reply:
Probably!